src/test/scala/za/co/absa/hyperdrive/trigger/testdata/LocalDatabaseConfig.scala

```diff
@@ -0,0 +1,29 @@
+
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.hyperdrive.trigger.testdata
+
+import org.scalatest.{BeforeAndAfterAll, Suite}
+
+trait LocalDatabaseConfig extends BeforeAndAfterAll { this: Suite =>
+
+  override def beforeAll(): Unit = {
+    System.setProperty("db.driver", "org.postgresql.Driver")
```

Comment:
Is it possible to use application properties file instead of this ? (I mean, without this class).

Reply:
Not entirely. From the spring side, I can use `@TestPropertySource` to define another properties file to use than the default one. However, slick obviously doesn't know about this. By default, Slick loads any configuration file with name `application.conf`, `application.json` or `application.properties` which is on the classpath. (See `JdbcBackend.forConfig`, `DefaultConfigLoadingStrategy.parseApplicationConfig`, `SimpleIncluder.fromBasename`). 

An alternative properties file can be specified with the system properties `config.resource`, `config.file` or `config.url` (which are unfortunately very general names, could easily be conflicted imo). So I could extract the three properties to a separate file, but I would still have to define the above mentioned system property.